src/Symfony/Bundle/TwigBundle/composer.json

```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": "^7.1.3",
-        "symfony/config": "^4.2|^5.0",
```

Comment:
this requirement should be kept, as we do depend on it (and we should not rely on the transitive dependency coming from elsewhere)